BackofficeBundle/Resources/views/layout.html.twig

```diff
@@ -84,6 +84,6 @@
             pageSetUp();
         });
     </script>
-
+    {{ tinymce_init() }}
```

Comment:
I override script.html.twig by extending bundle. So tinmce_init() will just load js library fo tiny, but not really init it.  initTinyMCE(); is in widgetChannel.coffee.